docs/chaos-engineering/chaos-faults/aws/lambda-update-function-timeout.md

```diff
@@ -0,0 +1,171 @@
+```
+
+- If you change the secret key name (from `cloud_config.yml`), update the `AWS_SHARED_CREDENTIALS_FILE` environment variable value on `experiment.yaml` with the same name.
+
+## Default Validations
+
+:::info
+
+- The Lambda function should be up and running.
+
+:::
+
+## Experiment Tunables
+
+<details>
+    <summary>Check the Experiment Tunables</summary>
+    <h2>Mandatory Fields</h2>
+    <table>
+      <tr>
+        <th> Variables </th>
+        <th> Description </th>
+        <th> Notes </th>
+      </tr>
+      <tr>
+        <td> FUNCTION_NAME </td>
+        <td> Function name of the target lambda function. It support single function name.</td>
+        <td> Eg: <code>test-function</code> </td>
+      </tr>
+      <tr>
+        <td> FUNCTION_TIMEOUT </td>
+        <td> Provide the value of function timeout in seconds.</td>
+        <td> The minimum value is 1s and maximum upto 15mins that is 900s </td>
+      </tr>
+      <tr>
+        <td> REGION </td>
+        <td> The region name of the target instance</td>
+        <td> </td>
+      </tr>
+    </table>
+    <h2>Optional Fields</h2>
+    <table>
+      <tr>
+        <th> Variables </th>
+        <th> Description </th>
+        <th> Notes </th>
+      </tr>
+      <tr>
+        <td> TOTAL_CHAOS_DURATION </td>
+        <td> The total time duration for chaos insertion (sec) </td>
+        <td> Defaults to 30s </td>
+      </tr>
+      <tr>
+        <td> CHAOS_INTERVAL </td>
+        <td> The interval (in sec) between successive instance termination.</td>
+        <td> Defaults to 30s </td>
+      </tr>
+      <tr>
+        <td> SEQUENCE </td>
+        <td> It defines sequence of chaos execution for multiple instance</td>
+        <td> Default value: parallel. Supported: serial, parallel </td>
+      </tr>
+      <tr>
+        <td> RAMP_TIME </td>
+        <td> Period to wait before and after injection of chaos in sec </td>
+        <td> Eg. 30 </td>
+      </tr>
+    </table>
+</details>
+
+## Experiment Examples
+
+### Common and AWS specific tunables
+
+Refer the [common attributes](../common-tunables-for-all-experiments) and [AWS specific tunable](./aws-experiments-tunables) to tune the common tunables for all experiments and aws specific tunables.
+
+### Function Name
+
+It injects the lambda function timeout chaos for the given function name using `FUNCTION_NAME` Env.
+
+Use the following example to tune this:
+
+[embedmd]:# (./static/manifests/lambda-update-function-timeout/function-name.yaml yaml)
+```yaml
+# contains the function name value for timeout chaos
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  chaosServiceAccount: litmus-admin
+  experiments:
+  - name: lambda-update-function-timeout
+    spec:
+      components:
+        env:
+        # provide the function name for timeout chaos
+        - name: FUNCTION_NAME
+          value: 'chaos-function'
```

Comment:
Similarly in this as well, we do not have any default value for `FUNCTION_TIMEOUT`